### PR TITLE
fix(entities): correct unavailable state

### PR DESCRIPTION
**Pull request by CFenner**
When the camera is not available these calls fail with an exception causing the entities not go into `unavailable` state.

Bildschirmfoto 2022-10-21 um 13 28 56

Adding `try/catch` in `select` and `camera` seems to fix this for most of the cases, `button` and `number` are still marked as `available`.

Related to #221

### CODE REVIEW

File: custom_components/tapo_control/select.py [hunk 1]
```diff
@@ -85,13 +85,17 @@ def __init__(self, entry: dict, hass: HomeAssistant, config_entry):
...
-        self._attr_current_option = await self._hass.async_add_executor_job(
-            self._controller.getLightFrequencyMode
-        )
+        try:
+            self._attr_current_option = await self._hass.async_add_executor_job(
+                self._controller.getLightFrequencyMode
+            )
+        except Exception:
+            self._attr_state = STATE_UNAVAILABLE
+            self._attr_current_option = None
```
Comment (CFenner) [inline custom_components/tapo_control/select.py:94]:
```suggestion
```

**Review by JurajNyiri — COMMENTED** [review]
We can remove all async_update functions in entities they are no longer needed

**Review by JurajNyiri — CHANGES_REQUESTED** [review]
See comment above

**Comment by JurajNyiri** [comment]
Thank you!!

**Comment by CFenner** [comment]
I haven't handed your comment yet, just solved the merge conflict. Was about to do that.

**Comment by JurajNyiri** [comment]
My bad, thank you!

**Pull request merged**